.github/workflows/tests.yml

```diff
@@ -2,13 +2,10 @@ name: Tests
 
 on:
   pull_request:
+  merge_group:
```

Comment:
Shouldn't we add that to other files like https://github.com/meilisearch/meilisearch-php/blob/main/.github/workflows/meilisearch-beta-tests.yml